pagopa/io-app
# chore(IT Wallet): [SIW-1759] Consider integrity check preparation

## Short description
This PR adds a new flag within the issuance store, saving the integrity check preparation result. In addition, a check is added in a machine actor in the eID obtainment flow

## List of changes proposed in this pull request
- Added `integrityServiceReady` flag
- Saved the result of `ensureIntegrityServiceIsReady` inside the store
- Added a check inside createEidIssuanceActorsImplementation to prevent the user without the preparation ready to proceed with the integrity check

## How to test
Open the app, and obtain the eID. You can simulate the polling by manipulating the polling function result or manipulating the store. The timeout is set to 10 seconds.

> [!NOTE]  
> This PR must be tested with an Android device since I don't own one. This PR fixes an issue related to some Android devices that weren't ready when doing the integrity check.
